### ISSUE TITLE
Cannot "make" PDF

### ISSUE THREAD

**Issue by lacabra** [unicef/publicgoods-toolkit#65]
In my local MacOS environment, after having installed [MacTex](https://www.tug.org/mactex/) (which provides the LaTEX system that pandoc depends on), I get the following error when running `make` to generate the PDF:
```bash
Error producing PDF.
! Argument of \paragraph has an extra }.
<inserted text> 
                \par 
l.1636 \ttl@extract\paragraph

make: *** [pdf] Error 43
```

Before I attempt to troubleshoot this, it would be good to understand the details of the system where @kfogel has run this successfully.

**Comment by kfogel** [comment]
Hey @lacabra.  My system is a Debian GNU/Linux recently-updated `testing` distribution, with Pandoc 2.9.2.1:

```
$ pandoc --version
pandoc 2.9.2.1
Compiled with pandoc-types 1.20, texmath 0.12.0.2, skylighting 0.8.5
```

What version of `pandoc` do you have, what version of MacTex, and what exact commit of the DPG toolkit tree were you trying to build?

From the error above, it looks like maybe pandoc's internal conversion from Markdown->LaTeX is generating invalid LaTeX? (Hence the syntax error about an extra "}", which presumably is in the (La)TeX input to MacTex.)

**Comment by lacabra** [comment]
Thanks @kfogel. I was using an older version it seems:

```bash
$ pandoc --version
pandoc 2.2.1
Compiled with pandoc-types 1.17.4.2, texmath 0.11, skylighting 0.7.1
```

I upgraded to the latest available in my system:
```bash
$ pandoc --version
pandoc 2.16.2
Compiled with pandoc-types 1.22.1, texmath 0.12.3.2, skylighting 0.12.1, citeproc 0.6, ipynb 0.1.0.2
```

And unfortunately, now I am getting a different error, which I am investigating:
```bash
$ make
pandoc: pdflatex: createProcess: posix_spawnp: illegal operation (Inappropriate ioctl for device)
make: *** [pdf] Error 1
```

I confirm that I'm on the latest commit of the `main` branch: e79a27d

**Comment by lacabra** [comment]
The previous error is a very cryptic one to say that `pdflatex` is [not installed in the system](https://github.com/jgm/pandoc/issues/7570#issuecomment-958748552). That's because `MacTex` is bundled as a 4GB file, which when installed takes more than 10GB, which is a lot! So I had uninstalled after seeing it was not working for me.

I managed to make it work with the following:

```bash
brew install pandoc
brew install basictex
sudo tlmgr update --self
sudo tlmgr install titlesec
```

Now it works just fine, closing (and I will document this elsewhere for future reference).
